### ISSUE TITLE
App makes the event unconfirmed by default when trying to edit it

### ISSUE THREAD

**Issue by Kalytis** [Etar-Group/Etar-Calendar#993]
When I edit an event, just trying to add an alarm to it, there is the "Participate?" box that shows up, which I never use. But if I don't check anything, it also marks the event as unconfirmed, which messes up my other calendar apps.

There really should be a way to confirm the event by default, or to remove this box when we don't use it.

Thanks by advance !

**Comment by c64bob** [comment]
I agree - events will be marked as unconfirmed when editing them by default. Every time I edit an event, I need to select "yes" under the "participate" box. This setting is not being saved properly.

Please let us chose a default for participation, or remove the box.

Thank you
